### DISCUSSION TITLE
Loading screen tips file location?

### DISCUSSION THREAD

**Discussion by SgtCookieBobble** [nzp-team/nzportable#696] [General]
Been tinkering with .txt files since some have grammatical errors as I like to make the descriptions seem official (like replacing or adding remade map's original descriptions.) Does anyone know where the loading screen tips are located, or are they deep in the code that requires a program to modify?

**Comment by Kempferi** [comment]
Yes, those are script based. This code can be found at https://github.com/nzp-team/quakec in the main.qc file from string LoadScreen_GiveTip

**Reply by SgtCookieBobble** [reply]
Much obliged.
